pixl_dcmd/src/pixl_dcmd/_database.py

```diff
@@ -14,9 +14,9 @@
 
 """Interaction with the PIXL database."""
 
-from decouple import config
+from decouple import config  # type: ignore [import-untyped]
 
-from core.db.models import Image
+from core.db.models import Image, Extract  # type: ignore [import-untyped]
```

Comment:
mm not sure why this is happening

Reply:
Ah woops, thought I reverted all of them